tmt/steps/provision/local.py

```diff
@@ -240,6 +255,11 @@ class ProvisionLocal(tmt.steps.provision.ProvisionPlugin[ProvisionLocalData]):
     .. note::
 
         Neither hard nor soft reboot is supported.
+
+    .. note::
+        The ``TMT_SCRIPTS_DIR`` variable is not supported in the
+        ``local`` provision. This solution has been provided as part
+        of issue #1853.
```

Comment:
We need a new issue for that as this will be closed with the pull request.
Filed here: https://github.com/teemtee/tmt/issues/4081

Reply:
Updated !